### DISCUSSION TITLE
Why don't we build buffalogs as if it is a plugable django supporting module?

### DISCUSSION THREAD

**Discussion by 0x0elliot** [certego/BuffaLogs#18] [Ideas]
I think such a tool would do the community well, especially if it is as well maintained as tools from intelowl have been. Why don't we look forward to building it that way or at least have plans to do so? 

I am not sure if we already do, but following the spirit of open-source, i would love to have the discussion continue over here.

**Comment by Lorygold** [comment]
BuffaLogs has become a Django Reusable App in the [Version 1.1.1](https://github.com/certego/BuffaLogs/releases/tag/1.1.1)
